style.css

```diff
@@ -406,3 +406,123 @@ body {
   margin-left: 15px;
   margin-right: 30px;
 }
+
+.contact-form {
+  width: 90%;
```

Comment:
- [ ] You have a small bug in your image URL or as for now all your images are not showing, your current image path is /image/.... but after my modification, it was displaying when I added a dot before the `/`, like `./image/...`, check the record below

![6](https://github.com/ariful04/Myportfolio/assets/77843050/e21a87e0-053c-4931-82b9-72c721e33eb6)